packaging/ansible-runner-service-project/project/roles/ovirt-ova-pack/files/pack_ova.py

```diff
@@ -44,15 +44,15 @@ def pad_to_block_size(file):
 def write_ovf(entity, ova_file, ovf):
     print("writing ovf: %s" % ovf)
     tar_info = create_tar_info(entity + ".ovf", len(ovf))
```

Comment:
worth aligning and add also into `create_tar_info()` - line 31 (`tarfile.TarInfo(name)`), no?

Reply:
@liranr23 is it relevant to the change that is introduced here? if it is, please rephrase

Reply:
Yes, I couldn't comment on line 31 in the `pack_ova.py`. We use there `tarfile` as well, I think it will be the best to use the format there as well, to have alignment in all `tarfile` usages here.

Reply:
In this case format can be used only when tarfile.TarInfo.tobuf() calling, not when create TarInfo.

Reply:
Yes, current implementation will work as you use `tobuf()` here with the format.
I took the idea of changing `TarInfo()` from: https://docs.python.org/3/library/tarfile.html#tarfile.TarFile

Reply:
OK, I think it's my bad. I messed up with `TarInfo` versus `TarFile` :)